src/scales/scale.linearbase.js

```diff
@@ -6,44 +6,65 @@ module.exports = function(Chart) {
 		noop = helpers.noop;
 
 	Chart.LinearScaleBase = Chart.Scale.extend({
+		// Following http://www.erichynds.com/blog/backbone-and-inheritance
+		constructor: function() {
+			// These two properties kept for backwards compatibility
+			var me = this;
+			Object.defineProperty(me, 'min', {
```

Comment:
hum, so it could break plugins that modify min and max?

Would not it be simpler to keep min/max (no dataRange) and create the object when calling the generator:

```javascript
var ticks = me.ticks = Chart.Ticks.generators.linear(numericGeneratorOptions, {
    min: me.min,
    max: me.max
});
```